### PR TITLE
Zalando workspace

### PR DESCRIPTION
**Pull request by MichalLic**

### CODE REVIEW

File: src/app/user/user.component.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+})
+export class UserComponent implements OnInit {
+
+  constructor() {
+  }
+
+  ngOnInit() {
+  }
+
+  onSubmit(f) {
```
Comment (parfienczyk) [inline src/app/user/user.component.ts]: what is 'f'? Please use longer param names

File: src/app/user/user.component.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            class="form-control"
+            required
+            #userPostCode="ngModel">
+          <span class="label label-danger" *ngIf="!userPostCode.valid && userPostCode.dirty">Please complete this field!</span>
+        </div>
+        <div class="form-group col-xs-7">
+          <label for="userPhone">Phone:</label>
+          <input
+            type="text"
+            name="userPhone"
```
Comment (parfienczyk) [inline src/app/user/user.component.html]: 'name' and 'id' should be not camelCase but for example  'name="user-phone"'

File: src/app/products/product/product.component.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+@import "../../../variables";
```
Comment (parfienczyk) [inline src/app/products/product/product.component.scss:1]: Try to use `@import "~variables";`

File: angular-cli.json [hunk 4]
```diff
@@ -18,9 +18,12 @@
...
       "prefix": "app",
       "styles": [
-        "styles.scss"
+        "styles.scss",
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
+        "../node_modules/owl.carousel/dist/assets/owl.carousel.css",
+        "../node_modules/font-awesome/css/font-awesome.css"
       ],
-      "scripts": [],
+      "scripts": ["../node_modules/jquery/dist/jquery.js"],
```
Comment (parfienczyk) [inline angular-cli.json:26]: What for we need jQuery?

Comment (MichalLic) [reply angular-cli.json:26]: it's necessary for owl carousel

Comment (parfienczyk) [reply angular-cli.json:26]: We can use pure/vanilla JS, for example:

> document.getElementById('owl-id-selector').owlCarousel({...HERE_SOME_OPTIONS...});

**Pull request merged**